res/cmake/funcs.cmake

```diff
@@ -0,0 +1,192 @@
+# public test functions
```

Comment:
I do not understand this file. It seems to be a copy of what's already in def.cmake

Reply:
I think the idea was to remove all the functions in "def.cmake" cause that file was getting a little hectic